package/Runtime/Scripts/Internal/NimbusAdUnit.cs

```diff
@@ -138,8 +138,8 @@ public sealed class NimbusAdUnit {
 
 		internal event DestroyAdDelegate OnDestroyIOSAd;
 
-		private void OnOnDestroyIOSAd() {
-			OnDestroyIOSAd?.Invoke();
+		private void DestroyIOSAd() {
```

Comment:
If this is an event it should be named OnOnDestroyIOSAd

Reply:
I renamed it because this is a method... Anyways, I removed it because this was not being reused and it is one line only.

Reply:
https://github.com/timehop/nimbus-unity/pull/21/commits/13470b73968ffa02a341f14c6e7a7a8285630bd5

Reply:
@brugg yeah I know it's a method. That is proper Unity + C# naming convention for methods that invoke events (fyi)